examples/undocumented/python/graphical/eigenfaces.py

```diff
@@ -69,9 +66,9 @@ def train(self, images, labels):
         self._labels = labels;
 
         #transform the numpe vector to shogun structure
-        features = RealFeatures(images)
+        features = sg.RealFeatures(images)
         #PCA
-        self.pca = PCA()
+        self.pca = sg.PCA()
```

Comment:
We started using factories instead of direct class instantiations
```suggestion
        self.pca = sg.transformer("PCA")
```